test/common/tap-cdp

```diff
@@ -66,13 +67,18 @@ if not cdp.get_browser_path():
     print('1..0 # skip web browser not found')
     sys.exit(0)
 
+# pass the address through a separate fd, so that we can see g_debug() messages (which go to stdout)
+(addr_r, addr_w) = os.pipe()
```

Comment:
shouldn't you close those at the end?

Reply:
Indeed I should immediately close the write end after passing it to the subprocess. Fixed, thanks for spotting!